src/style.css

```diff
@@ -0,0 +1,193 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-image: linear-gradient(120deg, #665521, #3d2b26);
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  min-height: 100vh;
+}
+
+header {
+  font-size: 1.5rem;
+}
+
+header,
+form {
+  min-height: 20vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+form input,
+form button {
+  padding: 0.5rem;
+  font-size: 2rem;
+  border: none;
+  background: #fff;
+}
+
+form button {
+  color: #3d2b26;
+  background: #fff;
+  cursor: pointer;
+  transition: all 0.3s ease;
+}
+
+form button:hover {
+  background: #3d2b26;
+  color: #fff;
+}
+
+.todo-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.todo-list {
+  min-width: 30%;
+  list-style: none;
+}
+
+.todo {
+  margin: 0.5rem;
+  background: #fff;
+  color: #000;
+  font-size: 1.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  transition: all 0.3s ease;
+}
+
+.todo li {
+  flex: 1;
+}
+
+.trash-btn,
+.complete-btn {
+  background: #b32323;
+  color: #fff;
+  border: none;
+  margin: 5px 2px  0 5px;
+  padding: 5px;
+  cursor: pointer;
+  font-size: 1rem;
+}
+
+.edit-btn {
+  background: #227;
+  color: #fff8db;
+  border: none;
+  margin: 5px 0  2px 5px;
+  padding: 5px;
+  cursor: pointer;
+  font-size: 1rem;
+}
+
+.complete-btn {
+  background: #97be6f;
+  margin: 5px 5px  2px 5px;
+  padding: 5px;
+  color: #000;
+}
+
+.todo-item {
+  padding: 0 0.5rem;
+}
+
+.fa-trash {
+  pointer-events: none;
+}
+
+.fa-check {
+  pointer-events: none;
+  color: #fff;
```

Comment:
You could create a variable for white color #fff this color appears 4 times in your css code;
![review](https://user-images.githubusercontent.com/80895497/130606735-c67b753c-15c0-4d4b-8013-16bb98653619.PNG)

Reply:
- Noted 🤝